File: hrms/src/main/java/com/kodlamaio/hrms/business/abstracts/PhoneNumberService.java
package com.kodlamaio.hrms.business.abstracts;

import com.kodlamaio.hrms.core.utilities.result.DataResult;
import com.kodlamaio.hrms.core.utilities.result.Result;
import com.kodlamaio.hrms.entities.conretes.Employer;
import com.kodlamaio.hrms.entities.conretes.PhoneNumber;

public interface PhoneNumberService {
	Result addEmployerPhone(int id, Employer employer ,PhoneNumber phoneNumber);
	public DataResult<PhoneNumber> isUpdate(int employerId,int exPhoneNumberId);
	public Result approve(int employerId, int phoneNumberId);
}

File: hrms-react/src/services/phoneNumberService.js
import axios from "axios";
import { toast } from "react-toastify";
export default class PhoneNumberService{

   isUpdate(employerId,exPhoneNumberId){
       return axios.get(`http://localhost:8080/PhoneNumber/isUpdate?employerId=${employerId}&exPhoneNumberId=${exPhoneNumberId}`)
   }

   updatePhoneNumber(employerId,phoneNumber){
       return axios.post(`http://localhost:8080/api/employer/updatePhoneNumber?employerId=${employerId}`,phoneNumber).then(function (response) {
          
        console.log(response.data.message)
        toast.success(response.data.message)
     })
     .catch(function (error) {
         toast.success(error.response.data)
         console.log(error.response.data)
     return error.response.data;
     });
   }

   approve(employerId,phoneNumberId){
        return axios.post(`http://localhost:8080/PhoneNumber/approve?employerId=${employerId}&phoneNumberId=${phoneNumberId}`).then(function (response) {
          
            console.log(response.data.message)
            toast.success(response.data.message)
         })
         .catch(function (error) {
             toast.success(error.response.data)
             console.log(error.response.data)
         return error.response.data;
   })
   }
   
}